test/test-login.js

```diff
@@ -35,18 +33,16 @@ describe('Logging in', () => {
         handlers.auth_callback[1](req, res, reject);
       });
 
-      const user_details_request = mock_api()
-        .get(`/users/${escape(user.auth0_id)}/`)
-        .matchHeader('Authorization', `JWT ${id_token}`)
-        .reply(200, user);
+      //const user_details_request = mock_api()
```

Comment:
Isn't `mock_api()`/nock working in the context of `withAPI()`?

Reply:
No - `mock_api`/nock mocks `http.request` and `http.clientRequest` to return a specified response. `withAPI` ensures that the call to `handlers.auth_callback` is bound to the namespace containing the API clients.